tests/tests/bitasset_tests.cpp

```diff
@@ -933,16 +933,11 @@ BOOST_AUTO_TEST_CASE( hf_935_test )
       int blocks = 0;
       auto mi = db.get_global_properties().parameters.maintenance_interval;
 
-      if( i == 2 ) // go beyond hard fork 890
+      if( i == 2) // go beyond hard fork 890 (which is the same as 935 BTW)
```

Comment:
I guess we can remove this test case directly if it's to test hf935 only. If it's used to test hf1270 as well but not documented, better rename the test case and update the document above.

Reply:
It is used to test 1270. Renamed the test and modified the comment above the method.